packages/analytics/dashboard-renderer/src/components/BarChartRenderer.vue

```diff
@@ -10,6 +10,7 @@
         :chart-options="options"
         legend-position="bottom"
         :show-annotations="chartOptions.showAnnotations"
+        tooltip-title=""
```

Comment:
@filipgutica tooltipTitle was removed in [your PR](https://github.com/Kong/public-ui-components/pull/1072) but I had to add it back because typecheck command was failing (I think it's a required prop?)

Reply:
Looks like it's filled in all other cases (all the other Sandbox demo pages). I think it's fine to bring that line back.